experimental/LieAlgebras/src/GapWrapper.jl

```diff
@@ -0,0 +1,68 @@
+function lie_algebra_struct_consts_gap(R::Ring, dynkin::Tuple{Char,Int})
+  @req is_valid_dynkin(dynkin...) "Input not allowed by GAP."
+  @req R == QQ "Works currently only for QQ." # TODO
+
+  GAPG = GAP.Globals
+
+  L = GAPG.SimpleLieAlgebra(GAP.Obj(string(dynkin[1])), dynkin[2], GAPG.Rationals)
+  dimL = GAPG.Dimension(L)
+  comm_table_L = Matrix{NTuple{2,Vector{Int}}}(
+    (GAPG.StructureConstantsTable(GAPG.Basis(L)))[1:dimL]
```

Comment:
```suggestion
    (GAPG.StructureConstantsTable(GAPWrap.Basis(L)))[1:dimL]
```
which is defined in `src/GAP/wrappers.jl` (there is a comment at the top of that file explaining why these wrappers exist and are usefu;)

We don't have to add all the functions you use here, but it would make sense to at least use those GAP wrappers that already exist. And perhaps add a few more, e.g. for `Dimension`.

Reply:
changed in 3d14ca13797f6f2ab910640674e7b0a35c5afc5a

I only added `Dimension` for now, since this seems widely applicable. Most other uses of `GAP.Globals` are special to Lie algebras. More can be added later if needed.